ocs_ci/ocs/acm/acm.py

```diff
@@ -369,12 +366,12 @@ def import_clusters_with_acm():
     """
     clusters_env = get_clusters_env()
     log.info(clusters_env)
-    kubeconfig_a = clusters_env.get("kubeconfig_cl_a_location")
-    kubeconfig_b = clusters_env.get("kubeconfig_cl_b_location")
+    kubeconfig_a = clusters_env.get("kubeconfig_location_c1")
+    kubeconfig_b = clusters_env.get("kubeconfig_location_c2")
     global cluster_name_a
-    cluster_name_a = clusters_env.get("cl_a_cluster_name")
+    cluster_name_a = clusters_env.get("cluster_name_1")
     global cluster_name_b
-    cluster_name_b = clusters_env.get("cl_b_cluster_name")
+    cluster_name_b = clusters_env.get("cluster_name_2")
```

Comment:
This doesn't look much scalable, but I understand that it was not introduced in this PR, but was already in the previous code. Also the usage of `global` is not nice in this context I think, but the same - it is not part of this PR.

Reply:
I am not a big fan of `global` as well but I think we should address it as a follow-up